add_zero_quantity/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+{
+    'name': "Add Zero Quantity",
+
+    'description':"Adding the zero quantity on account move line",
+
+    'author': "Odoo",
+    'category': 'Tutorials/AddZeroQuantity',
+    'installable': True,
+    'depends': ['l10n_in_edi', 'l10n_in_reports_gstr'],
```

Comment:
```suggestion
    'depends': ['l10n_in_reports_gstr'],
```
`l10n_in_reports_gstr` already depends on `l10n_in_edi` in `18.0`